package.json

```
@@ -21,14 +21,14 @@
     "license-checker": "^25.0.1",
     "lodash": "^4.17.21",
     "semver": "^7.3.5",
-    "spdx-license-ids": "^3.0.9",
+    "spdx-license-ids": "^3.0.10",
     "spdx-satisfies": "^5.0.1",
-    "yargs": "^17.0.1"
+    "yargs": "17.0.1"
```

This will cause tests to fail. More investigation required.
```suggestion
    "yargs": "17.1.0"
```

---

https://github.com/Brightspace/d2l-license-checker/pull/129